react/VideoTypes.tsx

```diff
@@ -10,11 +10,11 @@ export interface VideoPlayer extends VideoHTMLAttributes<HTMLVideoElement> {
   fallbackImageUrl?: string
   type?: string
   controlsType?: string
-  IconPlay?: FC<unknown>
-  IconPause?: FC<unknown>
-  IconFullscreen?: FC<unknown>
-  IconVolumeOn?: FC<unknown>
-  IconVolumeOff?: FC<unknown>
+  IconPlay?: ComponentType<unknown>
```

Comment:
Using ComponentType you will have problems with typescript if you inline a HTML, for example.

```tsx
function MyEnhancedVideo() {
  // Type 'Element' is not assignable to type 'ComponentClass<unknown, any> | FunctionComponent<unknown> | undefined'.
  return <Video src="" IconPlay={<button><svg>{/*svg goes here*/}</svg></button>}/>
}
```

I suggest using ReactNode.

Reply:
Interesting! We've been using `ComponentType` for all Slots, maybe we should reconsider it.

Reply:
I honestly think that's worth it. Since `ComponentType` best represents what we're expecting and is also a very intuitive name.